tools/nl/embeddings/requirements.txt

```diff
@@ -6,4 +6,9 @@ google-cloud-storage==2.15.0
 lancedb==0.6.8
 parameterized==0.8.1
 sentence-transformers==2.2.2
-torchvision==0.17.2
\ No newline at end of file
+torchvision==0.17.2
+# Downloading the named-entity recognition (NER) library spacy and the large EN model
+# using the guidelines here: https://spacy.io/usage/models#production
+# TODO: try using the large model
+-f https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.7.1/en_core_web_sm-3.7.1-py3-none-any.whl
+en_core_web_sm==3.7.1
```

Comment:
@shifucun - Needed to add this to the embeddings requirements to eliminate installing the nl_server requirements. Let me know if this is ok.

Reply:
I was even thinking get rid of the lib here and make a shared requirements.txt. Duplicating the lib makes the version diverge very easily. Saw an bug before due to this.